cmake/Modules/Packages/MBX.cmake

```diff
@@ -0,0 +1,118 @@
+# MBX v1.3.0+ support for MBX package
+# Based off of PLUMED.cmake
+
+# set policy to silence warnings about timestamps of downloaded files. review occasionally if it may be set to NEW
+if(POLICY CMP0135)
+  cmake_policy(SET CMP0135 OLD)
+endif()
+
+# for supporting multiple concurrent mbx installations for debugging and testing
+set(MBX_SUFFIX "" CACHE STRING "Suffix for MBX library")
+mark_as_advanced(MBX_SUFFIX)
+
+if(BUILD_MPI)
+  set(MBX_CONFIG_MPI "--enable-mpi")
+  set(MBX_CONFIG_CC  ${CMAKE_C_COMPILER})
+  set(MBX_CONFIG_CXX  ${CMAKE_CXX_COMPILER})
+  # set(MBX_CONFIG_CPPFLAGS "-I ${MPI_CXX_INCLUDE_PATH}")
+  # set(MBX_CONFIG_LIB "${MPI_CXX_LIBRARIES}")
+  # set(MBX_CONFIG_DEP "")
+else()
+  set(MBX_CONFIG_MPI "--disable-mpi")
+  set(MBX_CONFIG_CC  ${CMAKE_C_COMPILER})
+  set(MBX_CONFIG_CXX  ${CMAKE_CXX_COMPILER})
+  # set(MBX_CONFIG_CPPFLAGS "")
+  # set(MBX_CONFIG_LIB "")
+  # set(MBX_CONFIG_DEP "")
+
+
+
+endif()
+
+
+set(MBX_URL "https://github.com/paesanilab/MBX/archive/refs/tags/v1.3.0.tar.gz"
+  CACHE STRING "URL for MBX tarball")
+set(MBX_MD5 "7cfbf221f9c249c364c7f47769d0d768" CACHE STRING "MD5 checksum of MBX tarball")
+
+mark_as_advanced(MBX_URL)
+mark_as_advanced(MBX_MD5)
+GetFallbackURL(MBX_URL MBX_FALLBACK)
```

Comment:
Where is MBX_FALLBACK defined?
This would usually be pointing to the LAMMPS download server (so we can have a clean copy, even if for some reason GitHub won't be accessible or changes the downloads again).
```suggestion
```